ntp.yml

```diff
@@ -18,6 +18,7 @@
 barclamp:
   name: 'ntp'
   display: 'NTP'
+  description: 'Common NTP service for the cluster'
```

Comment:
Can we take this as an opportunity to get a better string? :-) Either "NTP Management" like for many other barclamps or something else.